Neos.Flow/Configuration/Testing/Settings.yaml

```diff
@@ -49,8 +49,9 @@ Neos:
                 logFileURL: '%FLOW_PATH_DATA%Logs/I18n_Testing.log'
                 severityThreshold: '%LOG_DEBUG%'
 
-
-
+    mvc:
+      routes:
+        'Neos.Flow': true
```

Comment:
This will break functional tests in the Neos distribution

Reply:
Dropped the commit again…